main/RFXNames.cpp

```diff
@@ -2759,6 +2760,11 @@ bool GetLightCommand(
 			cmd=Limitless_SetRGBColour;
 			return true;
 		}
+		else if (switchcmd == "Set HexRGBColor")
+		{
+			cmd = Limitless_SetHEXColour;
```

Comment:
Limitless_SetHEXColour is added by 'someone' but not used (also not needed as we can use 'Limitless_SetRGBColour'
I see internally you also use 'RGB' in the variable/function
(and i think also pLed->Value which is a uint8 i thought